llmstudio/engine/providers/provider.py

```diff
@@ -268,28 +268,6 @@ def join_chunks(self, chunks, request):
                 ):
                     function_call_arguments += chunk.get("arguments")
 
-            chunk = ChatCompletion(
```

Comment:
will this break other providers like openai, azureopen ai, etc?

Reply:
This was included for debugging purposes and does not impact the code's behavior.